src/odemis/driver/xt_client.py

```diff
@@ -21,20 +21,21 @@
 """
 from __future__ import division, print_function
 
-import base64
 import logging
 import threading
 import time
 from concurrent.futures import CancelledError
 
-import msgpack
-import msgpack_numpy as m
-from Pyro5.api import Proxy
+import msgpack_numpy
+from Pyro5.api import Proxy, config
 
 from odemis import model
 from odemis import util
 from odemis.model import CancellableThreadPoolExecutor, HwError, isasync, CancellableFuture
 
+config.SERIALIZER = 'msgpack'
```

Comment:
`config` is a very generic term. It's not very convenient to have it in the main namespace. Moreover, it's only used here. So I'd suggest to explicitly write `Pyro5.api.config`.